dockerfiles/Dockerfile-snapp-test-transaction

```diff
@@ -0,0 +1,38 @@
+ARG deb_codename=stretch
+FROM debian:${deb_codename}-slim
+
+# Run with `docker build --build-arg deb_version=<version>`
+ARG deb_version
+ARG deb_codename=stretch
+ARG deb_release=alpha
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN echo "Building image with version $deb_codename $deb_release $deb_version"
+
+COPY scripts/archive-entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
+
+# Dependencies
+RUN apt-get -y update \
+    && apt-get -y upgrade \
+    && apt-get -y install \
+        procps \
+        curl \
+        jq \
+        libjemalloc-dev \
```

Comment:
Remove libjemalloc-dev and libpq-dev, rely on the proper dependencies for the debian version instead

Reply:
I added `libjemalloc-dev` to get the proper `libjemalloc`. Without that, I got the unfound .so file errors (different .so's for stretch and buster).

Is there some other way to accomplish that?

Reply:
I think I had tried removing `libpq-dev` and got an error, but not certain of that.

Reply:
changed the `Depends:` and removed these, testing now